shovel-ts/test/index.test.ts

```diff
@@ -0,0 +1,72 @@
+import { expect, test } from "bun:test";
+import { makeConfig } from "../src/index";
+import type { EthSource, Table, Integration } from "../src/index";
+
+test("XXX", () => {
+    const transfersTable: Table = {
+        name: "transfers",
+        columns: [
+            { name: "from", type: "bytea" },
+            { name: "to", type: "bytea" },
+            { name: "value", type: "numeric" },
+        ]
+    };
+    const mainnet: EthSource = {
+        name: "mainnet",
+        url: "https://ethereum.publicnode.com",
```

Comment:
might be a nice affordance to take in `envVar('RPC_URL')` or `http('https...')` so people don't have to know about the specifics of how shovel.config.json is expected to be formatted

Reply:
I like this idea. One thing tho is that the name of an ethereum source is important since it is used throughout the database. How did you imagine that working with your suggestion?

Reply:
i imagine you'd pass the EthSource to the integrations directly and they would take out the name property from it so you have type safety (assuming you don't make a new eth source every time you define a new integration)

Reply:
Brilliant!

Reply:
I suppose the `makeIntegration` function you suggested could take a `EthSource[]`. Ya?

Reply:
ya!

Reply:
I'm going to save this for another PR